src/Schema.php

```diff
@@ -90,6 +93,14 @@ public static function ensureFrontControllerAccess() {
     static::createOrPrependFile(ABSPATH . '.htaccess', $template, "\n");
   }
 
+  /**
+   * Set Cache-Control header with a TTL of 1 year for all assets.
+   */
+  public static function ensureAssetsCacheControl() {
+    $template = file_get_contents(Plugin::getBasePath() . '/conf/.htaccess.assets.cache.control');
```

Comment:
Can we remove the "control"? It's too specific.  Sorry for the change of mind 🙈 
```suggestion
    $template = file_get_contents(Plugin::getBasePath() . '/conf/.htaccess.assets.cache');
```
The block delimiter can change accordingly into `core-standards:assets-cache`